cmd/gossamer/main.go

```diff
@@ -61,9 +65,26 @@ func main() {
 	}
 }
 
+func AddVerbosity(logger log.Logger, ctx *cli.Context) error {
+	lvl, err := log.LvlFromString(ctx.String("verbosity"))
+	if err != nil {
+		return err
+	}
+	handler := logger.GetHandler()
+	logger.SetHandler(log.LvlFilterHandler(lvl, handler))
+
+	return nil
+}
+
 // gossamer is the main entrypoint into the gossamer system
 func gossamer(ctx *cli.Context) error {
 	srvlog := log.New(log.Ctx{"blockchain": "gossamer"})
```

Comment:
This line stops everything from working unfortunately. To understand why we need to know that log15 automatically initializes a logger for us, inside its `init()` function: https://github.com/ChainSafe/log15/blob/b92444566a9f836e6ff88bba74cbb98b96e203d6/root.go#L17

The fix here is to replace it with `log.Root()` to fetch this global instance, instead of initializing a new one.